frontend/src/app/shared/components/header/header.component.sass

```diff
@@ -61,7 +61,9 @@
       cursor: pointer
 
 .main-area
-    padding-left: 10%
+    position: absolute
+    left: 50%
+    transform: translateX(-50%)
```

Comment:
Can we go without position:absolute?
Make left/right block the same width (in CSS) and use display:flex